package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "csurf",
   "description": "CSRF token middleware",
-  "version": "1.10.0",
+  "version": "1.10.1",
```

Comment:
Please remove this version bump from the PR, as we used npm to make the new releases and tags, which bumps and commits this for us.

Reply:
Let me create the same pr in few minutes

Reply:
Nice, thank you. I assume you are different from the OP here? I will at least give OP a chance to work on their PR here so I don't rob them of making a good contribution

Reply:
sure. As I said, all what I need - to get changes. If it will be Jared - np.